site/docs/Tutorial-BasicTypes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Basic Types
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/basic-types/
+next: /graphql-js/passing-arguments/
+---
+
+In most situations, all you need to specify the types for your API is the GraphQL schema language, taken as an argument to the `buildSchema` function.
+
+The GraphQL schema language supports the scalar types of `String`, `Int`, `Float`, `Boolean`, and `ID`, so you can use these directly in the schema you pass to `buildSchema`.
```

Comment:
We have it in this section: https://github.com/graphql/graphql.github.io/blob/30c5738b927505f162ce585d26727b9630d63b4f/site/docs/Learn-Schema.md#enumeration-types